### DISCUSSION TITLE
想请教如何只编译一个小章节并生成pdf供阅览

### DISCUSSION THREAD

**Discussion by 8electron** [tuna/thuthesis#908] [Q&A] [answered]
例如我希望只build data/chap04.tex and generate the corresponding pdf file. 一有修改想看正文就需要编译全文的话，需要的时间太长了。
已经读完并搜索了QA没有相似讨论（难道是这个问题太笨了吗？）我用的是TeX Live+VS Code。
感谢！

**Comment by zepinglee** [comment, marked as answer]
在主文档中注释掉其他章节。

LaTeX 还有一个 `\includeonly` 命令，但其实用起来不方便。

**Reply by 8electron** [reply]
感谢！
用`\iffalse` `\fi`可以注释掉一大段

**Reply by zepinglee** [reply]
`\iffalse` ... `\fi` 并不是严格意义上的注释。TeX 还是会处理其中的内容，遇到错误还会报错。总之不如 `%` 注释。

**Reply by 8electron** [reply]
好滴好滴，那就只能一行一行地注释掉了
